File: testUDPscripts/send_req_once.lua
local socket = require("socket")
local json = require("dkjson")

local udp = socket.udp()
udp:setpeername("0.0.0.0", 10000)

udp:send(json.encode({ auth = 'password', note = 'testing' }))
udp:close()

File: testUDPscripts/send_req_once.rb
require 'socket'
require 'digest'
require 'json'

PORT = 10000

udp = UDPSocket.new

udp.setsockopt(Socket::SOL_SOCKET, Socket::SO_BROADCAST, true)
udp.send( { auth: Digest::SHA1.hexdigest(File.readlines('secret.dat')[0]), note: 'testing' }.to_json, 0, '<broadcast>', PORT )
udp.close